components/Internship/NextSteps.js

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+import React, {useState} from 'react';
+
+import styles from '../../styles/components/Internship/NextSteps.module.scss';
+
+function NextSteps(props) {
+    const { image, name, info } = props;
+    const [isFlipped, setFlipped] = useState(true);
+    return (
+        <div className={styles['committee-card']} onClick={() => setFlipped(!isFlipped)}>
+            <div className={`${styles['next-steps-card-inner']} ${styles[isFlipped ? null : 'is-flipped']}`}>
+                <div className={styles['next-steps-card-face']}>
+                    <Image
+                        src={image}
+                        alt={`${name}'s card motif`}
+                        layout='fill'
+                    />
+                </div>
+                <div className={`${styles['next-steps-card-face']} ${styles.back}`}>
```

Comment:
This should probably be a `p` tag, or at least something else!

Reply:
this shouldn't be a `<p>`, since it's for the entire card face - not just the text
I can however, add a `<p>` for the next line before and after the actual info

Reply:
I guess my question is more, can we remove one level of nesting here? There's no styling on `<p>{info}</p>`, so we should be able to make the entire card face (the wrapping `<div>`) a paragraph. There's nothing else in the card face other than the paragraph, right?

Reply:
This would have been applicable a while ago, but in the interests of clarity I've added a heading before the `<p>` with the committee name